pkg/controller/appinst_api.go

```diff
@@ -1176,6 +1179,7 @@ func (s *AppInstApi) createAppInstInternal(cctx *CallContext, in *edgeproto.AppI
 			if cloudletCompatibilityVersion >= cloudcommon.CRMCompatibilitySharedRootLBFQDN {
 				// no external access to AppInst, no need for URI
 				in.Uri = ""
+				in.StaticUri = ""
```

Comment:
All these instances of `in.StaticUri = in.Uri` can be consolidated to one instance right before calling`s.store.STMPut(stm, in)`.